src/css/bootstrap/custom.scss

```diff
@@ -4,7 +4,7 @@ $brand-primary: #5B34B6;
 $body-color: #5B34B6;
 $link-color: #5B34B6;
 
-$font-size-base: 1rem * 1.5; // Assumes the browser default, typically `16px`
+$font-size-base: 1rem * 2.5; // Assumes the browser default, typically `16px`
```

Comment:
It seems odd that base is changed but not the others, since lg/sm/xs are in relation to base.